.github/workflows/generate-ref-docs.yaml

```diff
@@ -0,0 +1,117 @@
+name: Deploy Reference Docs
+
+on:
+  workflow_call:
+    inputs:
+      buildsystem:
+        description: 'Build system to use. Currently only supports `gradle` and `maven`'
+        type: string
```

Comment:
Can you please if `choice` is supported for this case? if not, let's convert the input to lower case so we don't care about similar inputs like: `maven`, `Maven`, and `MAVEN`, etc...

Reply:
done.